kernel/redFlags.ml

```diff
@@ -0,0 +1,114 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+open Util
+open Names
+
+let all_opaque = TransparentState.empty
+
+(* [r_const=(true,cl)] means all constants but those in [cl] *)
+(* [r_const=(false,cl)] means only those in [cl] *)
+(* [r_delta=true] just mean [r_const=(true,[])] *)
```

Comment:
This is outdated, the boolean in r_const in the comment is now handled by the Pred modules in the transparent state fields, and the meaning of delta changed in https://github.com/coq/coq/pull/14810

Reply:
OK, I removed the comment.